wrappers/dotnet/README.md

```diff
@@ -7,6 +7,10 @@ The wrapper is designed to be platform independent and targets the .NET Standard
 
 Pull requests welcome!
 
+### Documentation
+
+Documentation for the .NET wrapper is available [here](https://srottem.github.io/indy-sdk/wrappers/dotnet/docs/index.html).
```

Comment:
@srottem @vimmerru should we move it to hyperledger.github.io ?

Reply:
I asked about hosting the docs a couple of months back on Rochetchat. I never got an answer so pursued a solution myself. I have no objection to it being moved but I do want it live somewhere it can be linked to from the project page.

Reply:
@vimmerru @ashcherbakov ^

Reply:
For me it looks much better to store documentation and wrapper code in one repo. I don't see problems in merging of  dotnet/docs/* to main repo.

Reply:
The way I've approched it on my repo is to segregate the generated documentation into a separate branch named gh-pages so that it doesn't pollute the codebase but is available as github.io pages.  The project has code for *generating* the documentation already under dotnet/indy-sdk-dotnet-doc.  Just an FYI.

Reply:
I've added the generated docs to the branch and updated the readme.md to link to the hyperledger.github.io page.  The indy-sdk repo will need to be configured to enable Github Pages on the master branch.